src/types.js

```diff
@@ -19,18 +79,46 @@ export const TYPES = {
     name: 'Community Guidelines',
     fileName: 'community_guidelines'
   },
+  customerPrivacyPolicy: {
+    name: 'Costomer Privacy Policy',
+    fileName: 'costumer_privacy_policy'
+  },
+  DMCA: {
+    name: 'DMCA',
+    fileName: 'dmca'
+  },
+  EULA: {
```

Comment:
In favor of merging with `tos`, unless I misunderstand and there is a difference.